src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -37,9 +41,44 @@ public static JsonSerializerSettings GetSerializerSettings()
             return settings;
         }
 
+        /// <summary>
+        /// Customises the given serializer settings
+        /// using provider configuration.
+        /// Can be used by any provider, allowing the users to use a standard set of configuration attributes.
+        /// </summary>
+        /// <param name="set"></param>
+        /// <param name="config"></param>
+        /// <returns><see cref="JsonSerializerSettings" /></returns>
+        public static JsonSerializerSettings UpdateSerializerSettings(JsonSerializerSettings set, IProviderConfiguration config)
+        {
+            if (config.Properties.ContainsKey("UseFullAssemblyNames"))
+            {
+                bool useFullAssemblyNames = false;
+                var UseFullAssemblyNamesValue = config.Properties["UseFullAssemblyNames"];
+                bool.TryParse(UseFullAssemblyNamesValue, out useFullAssemblyNames);
+                if (useFullAssemblyNames)
```

Comment:
should technically be `if (bool.TryParse(UseFullAssemblyNamesValue, out useFullAssemblyNames) && useFullAssemblyNames)`
Currently the behavior is similar, I'm just worried about maintainability, where either the default can change or someone copy/pastes this code for a new setting where the default is not `false`